### PR TITLE
docs: add GitPOAP section to README

### PR DESCRIPTION
**Pull request by SakiAoiaoi**
For new contributors like me,it would be more useful to see a GitPOAP explanation in the README.

This file is the same as my pull request here:https://github.com/celestiaorg/celestia-core/pull/1156
It might be helpful for beginners to include this in the READM.

close https://github.com/celestiaorg/celestia-core/issues/1155

### CODE REVIEW

**Review by cmwaters — APPROVED** [review]
Thanks

File: README.md [hunk 1]
```diff
@@ -91,6 +91,10 @@ Releases are formatted: `v<CELESTIA_CORE_VERSION>-tm-v<TENDERMINT_CORE_VERSION>`
 For example: [`v1.4.0-tm-v0.34.20`](https://github.com/celestiaorg/celestia-core/releases/tag/v1.4.0-tm-v0.34.20) is celestia-core version `1.4.0` based on CometBFT `0.34.20`.
 `CELESTIA_CORE_VERSION` strives to adhere to [Semantic Versioning](http://semver.org/).
 
+## GitPOAP
+
+Once your pull request is merged to an eligible celestiaorg GitHub repository, you will be rewarded the ability to mint a [GitPOAP](https://www.gitpoap.io). You can check which repositories are eligible at [gitpoap/celestiaorg](https://www.gitpoap.io/gh/celestiaorg)
```
Comment (rootulp) [inline README.md:96]: [nit]

```suggestion
This repo is eligible for [GitPOAP](https://www.gitpoap.io). You can see the remaining eligible repositories at [gitpoap/celestiaorg](https://www.gitpoap.io/gh/celestiaorg).
```

**Review by rootulp — COMMENTED** [review]
I'm indifferent on whether we should include this in the README.

**Comment by rootulp** [comment]
Updated the PR title to conform to https://www.conventionalcommits.org/en/v1.0.0/

**Comment by SakiAoiaoi** [comment]
Thank you for putting a URL about PR title.
PR title has rules which keeps the commit history organized and ready for future automated tools.....!
I am so impressed with the specifications and your kindness. Thanks again..

**Review by rootulp — APPROVED** [review]

**Review by cmwaters — APPROVED** [review]

**Pull request merged**